spec/services/rapid_ready_for_decision/rrd_special_issue_manager_spec.rb

```diff
@@ -12,7 +12,7 @@ def form526_hash(form)
 
   def filter_disabilities(form)
```

Comment:
Suggestion: use a more descriptive method name:
```suggestion
  def supported_disabilities(form)
```